assets/sass/modules/_enroll.scss

```diff
@@ -33,11 +33,19 @@
 
 .enroll-factor-row {
   margin-bottom: 30px;
+  display: flex;
+  align-items: center;
 
   .enroll-factor-description {
     overflow: hidden;
   }
 
+  .enroll-factor-label {
+    display: flex;
+    justify-content: space-between;
+    min-width: 250px;
```

Comment:
do `width: 100%` or `flex-shrink: 0` solve this? might be less brittle than the `min-width`

Reply:
both doesn't help. I dont think the widget has any responsive css.

Container min-width is 400px.

Reply:
we have some responsive css for the white background, but not for labels or icons...